@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/XEH_preInit.sqf

```diff
@@ -122,7 +122,7 @@ fza_ah64_firekeypressed = 0;
 fza_ah64_overallticker = 0;
 fza_ah64_locktargstate = 0;
 fza_ah64_irjammer = 0;
-fza_ah64_salvofired = 0;
+fza_ah64_salvofired = 1;
```

Comment:
I don't think this is the correct way. To me it looks like the fired event handler order is incorrect. Maybe try in `extendedEventHandlers.hpp` changing the fired eventhandler from

```cpp
firedBIS = "_this call fza_fnc_eventFired; _this call fza_fnc_fxMuzzle; fza_ah64_salvofired = fza_ah64_salvofired + 1;";
```

to

```cpp
firedBIS = " _this call fza_fnc_fxMuzzle; fza_ah64_salvofired = fza_ah64_salvofired + 1;_this call fza_fnc_eventFired;";
```